### ISSUE TITLE
Handle directories as input

### ISSUE THREAD

**Issue by CandyAngel** [jhthorsen/app-podify#4]

**Comment by jhthorsen** [comment]
Would you also add `-r` and `--recursive` ?

Maybe something like this..?

```
option bool => recursive => 'Podify all files recursively in a folder', alas => 'r';
app {
  my ($self, @paths) = @_;

  if (-d $paths[0] and $self->recursive) {
   my $dir = $paths[0];
   @paths = ();
   File::Find::find(sub {push @paths, $File::Find::name if /\.pm$/}, $dir);
  }
}
```

**Comment by CandyAngel** [comment]
Implemented in ae572a9cd718164570df07e1bb3bf847c1372082
